lib/hardware_control/live_view_streaming/nokhwa_camera_stream_factory.dart

```diff
@@ -29,9 +30,16 @@ class NokhwaCameraStreamFactory extends LiveViewStreamFactory {
 
   @override
   Stream<LiveViewFrame> getStream() {
-    return rustLibraryApi
-      .nokhwaSetCameraCallback(cameraPtr: _cameraPointer, operations: [ImageOperation.cropToAspectRatio(3/2)])
-      .map((rawImage) => LiveViewFrame(rawRgbaData: rawImage.data, width: rawImage.width, height: rawImage.height));
+    return rustLibraryApi.nokhwaSetCameraCallback(cameraPtr: _cameraPointer, operations: [
+      ImageOperation.cropToAspectRatio(3 / 2)
+    ]).map((liveCameraFrame) {
+      StatsManagerBase.instance.addLiveViewFramesDroppedByCameraImplementation(liveCameraFrame.skippedFrames);
```

Comment:
This will be a number that's usually 0 then?

Reply:
Yes, unless something went "wrong" (aka Flutter app didn't respond anymore and Rust lib started to throw away frames to keep memory usage from building up)